src/otx/algorithms/anomaly/tasks/train.py

```diff
@@ -91,7 +90,8 @@ def train(
         ]
 
         if is_xpu_available():
-            callbacks.append(XPUCallback())
```

Comment:
If `XPUCallback` isn't used, how about removing it?

Reply:
I wanted to keep it just in case, but since the new approach works well, let's delete it